specification/gp-connect-access-record-fhir.yaml

```diff
@@ -33,79 +40,105 @@ info:
+    * is intended only for use by patient-facing applications
 
-    # Prerequisites
+    * is intended only for non-clinical use
 
-    # Technical
+    * can only be used where there is a legal basis to do so
 
-    The technical prerequisites are as follows:
+    Make sure you have a valid use case before you go too far with development. To do this, contact us. 
 
-    you must have access to the [Health and Social Care Network](https://digital.nhs.uk/services/health-and-social-care-network) (HSCN)
+    You must do this before you can go live (see Onboarding below).
 
-    you must be [Personal Demographics Service](https://digital.nhs.uk/services/demographics) (PDS)-compliant or capable of performing a PDS search via NHS Digital or a third-party provider
+    ### Prerequisites 
 
-    # Information governance (IG)
+    #### Information governance (IG)
+    
+    You must:
 
-    The IG prerequisites are as follows:
+    * be compliant with the GP Connect Direct Care API Information Governance Model
 
-    your organisation must be compliant with the [GP Connect Direct Care API Information Governance Model](https://github.com/nhsconnect/gpc-consumer-support/wiki/Information-Governance-(IG))
+    #### Clinical safety 
 
-    # Clinical safety
+    You must:
 
-    The clinical safety prerequisites are as follows:
-
-    you must have a clinical safety officer (CSO) who is responsible for[DCB0129](https://digital.nhs.uk/data-and-information/information-standards/information-standards-and-data-collections-including-extractions/publications-and-notifications/standards-and-collections/dcb0129-clinical-risk-management-its-application-in-the-manufacture-of-health-it-systems)and, if necessary,[DCB0160](https://digital.nhs.uk/data-and-information/information-standards/information-standards-and-data-collections-including-extractions/publications-and-notifications/standards-and-collections/dcb0160-clinical-risk-management-its-application-in-the-deployment-and-use-of-health-it-systems)(for more on clinical risk management, visit [Clinical risk management standards](https://digital.nhs.uk/services/clinical-safety/clinical-risk-management-standards))
-
-    If you are confident that you can meet the prerequisites, please express an interest with the GP Connect team (see 'Onboarding' below).
+    * have a clinical safety officer (CSO) who is responsible for DCB0129 and, if necessary, DCB0160. For more on clinical risk management, visit Clinical risk management standards
 
     ## Related APIs
-
+    
     The following APIs are related to this API:
+    
+    * [GP Connect Access Document - FHIR API](https://digital.nhs.uk/developer/api-catalogue/gp-connect-access-document-fhir) - use this API to retrieve documents from a patient’s registered GP practice, after you've found the relevant document references within Access Record returns, use GP Connect Access Document to retrieve the documents. All documents can be requested separately.
```

Comment:
Don't think this is relevant as can't be used for patient facing services

Reply:
removed